bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -314,11 +369,50 @@ func (p *packer) encodeTuple(t Tuple, nested bool) {
 // call Pack when using a Tuple with a FoundationDB API function that requires a
 // key.
 func (t Tuple) Pack() []byte {
-	p := packer{buf: make([]byte, 0, 64)}
+	p := newPacker()
 	p.encodeTuple(t, false)
 	return p.buf
 }
 
+// PackWithVersionstamp packs the specified tuple into a key for versionstamp operations
+func (t Tuple) PackWithVersionstamp() ([]byte, error) {
```

Comment:
The godoc descriptions for this function and `HasIncompleteVersionstamp` should probably specify what errors the functions might return.

Reply:
👍